app/code/Magento/User/Controller/Adminhtml/Auth.php

```diff
@@ -18,19 +19,28 @@
      * @var \Magento\User\Model\UserFactory
      */
     protected $_userFactory;
+    
+    /**
+     * @var \Magento\Backend\Helper\Data
+     */
+    protected $_backendHelper;
```

Comment:
It is actually a `backendDataHelper`. The `Backend\Helper` namespace has more `Helper` classes.

Reply:
Done.